package.json

```diff
@@ -299,6 +303,11 @@
                             "%containerApps.deleteConfirmation.ClickButton%"
                         ],
                         "default": "EnterName"
+                    },
+                    "containerApps.enableIngress": {
```

Comment:
Having ingress enabled is a property of an individual container app isn't it?

Reply:
Yes, you are correct 😊

This change would disable the command for all container apps. In this case it is probably best to show the command but do the check after a specific container app is chosen.

Reply:
This is a good use case for context values. We can apply a context value to note that ingress is enabled on those container apps for which it is the case, and show the "Disable Ingress" command only for those matching the context value.